stdlib/universe/contains.go

```diff
@@ -0,0 +1,104 @@
+package universe
+
+import (
+	"fmt"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/pkg/errors"
+	"regexp"
+)
+
+// Contains will test whether a given value is a member of the given set array.
+type ContainsFunc struct{}
+
+func (c *ContainsFunc) Type() semantic.Type {
+	return semantic.Invalid
+}
+func (c *ContainsFunc) PolyType() semantic.PolyType {
+	return semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"value": semantic.Tvar(1),
+			"set":   semantic.NewArrayPolyType(semantic.Tvar(2)),
```

Comment:
These should both reference Tvar(1), that way the value will always have to be the same types as the elements in the array.

Reply:
oh, that's cool.  I should have noticed that.